libs/upbound-provider-opentofu/config.jsonnet

```diff
@@ -0,0 +1,13 @@
+local config = import 'jsonnet/config.jsonnet';
+
+config.new(
+  'upbound-opentofu',
```

Comment:
The CI failure seems to point towards this one needing to be `upbound-provider-opentofu` (or the directory name should be `upbound-opentofu`).